$OpenBSD: patch-libalkimia_LibAlkimiaConfig_cmake_in,v 1.1 2017/01/12 16:18:55 zhuk Exp $
Unbreak CMake config file, allowing libalkimia to be found and used.
--- libalkimia/LibAlkimiaConfig.cmake.in.orig	Wed Jan 11 00:36:02 2017
+++ libalkimia/LibAlkimiaConfig.cmake.in	Wed Jan 11 00:36:08 2017
@@ -5,7 +5,9 @@ find_package(GMP REQUIRED)
 
 include("${CMAKE_CURRENT_LIST_DIR}/LibAlkimiaTargets.cmake")
 
-set_and_check( LIBALKIMIA_INCLUDE_DIR "@INCLUDE_INSTALL_DIR@")
+set_and_check( LIBALKIMIA_INCLUDE_DIR "@KDE4_INCLUDE_INSTALL_DIR@")
+
+set( LIBALKIMIA_INCLUDE_DIRS ${LIBALKIMIA_INCLUDE_DIR} ${GMP_INCLUDE_DIR} )
 
 set( LIBALKIMIA_LIBRARIES
   Alkimia::alkimia
